docs/source/getting_started.rst

```diff
@@ -3,150 +3,182 @@
+The following example runs an application inside the container and writes the output into a directory of the local files system,
+
+.. code-block:: console
+
+    docker run --rm -it --env-file .env \
+        -v "$(pwd):/workdir/external" \
+        ghcr.io/gammasim/simtools-prod:latest \
+        simtools-print-array-elements \
+        --array_element_list ./simtools/tests/resources/telescope_positions-North-utm.ecsv \
+        --export corsika --use_corsika_telescope_height \
+        --output_path /workdir/external/
 
 .. _InstallationForDevelopers:
 
 Installation for Developers
----------------------------
+===========================
 
 Developers install simtools directly from the GitHub repository:
 
-++++++++++++++++++++
-Clone simtools
-++++++++++++++++++++
-
 .. code-block:: console
 
     $ git clone https://github.com/gammasim/simtools.git
+    $ cd simtools
 
-++++++++++++++++++++
-Install dependencies
-++++++++++++++++++++
-
-Create a conda virtual environment with the simtools dependencies installed:
+Create a conda/mamba virtual environment with the simtools dependencies installed:
 
 .. code-block:: console
 
-    $ conda env create -f environment.yml
-
-    $ conda activate simtools-dev
-
+    $ mamba env create -f environment.yml
+    $ mamba activate simtools-dev
     $ pip install -e .
 
-++++++++++++++++++++++++++++++++
-Install CORSIKA and sim_telarray
-++++++++++++++++++++++++++++++++
+.. _CorsikaSimTelarrayInstallation:
 
-CORSIKA and sim_telarray are external tools to simtools and are used by several simtools applications.
-Follow the instruction provided by the CORSIKA/sim_telarray authors for installation.
-CTA users can download both packages from `here <https://www.mpi-hd.mpg.de/hfm/CTA/MC/Software/Testing/>`_ and install the package with e.g.:
+Installation of CORSIKA and sim_telarray
+========================================
 
-.. code-block:: console
+CORSIKA and sim_telarray are external tools to simtools and are required dependencies for many applications.
+Recommended is to use the Docker environment, see description in  `Docker Environment for Developers`_.
 
-    tar -czf corsika7.7_simtelarray.tar.gz
-    ./build_all prod5 qgs2 gsl
+For a non-Docker setup, follow the instruction provided by the CORSIKA/sim_telarray authors for installation.
+CTA users can download both packages from the `sim_telarray webpage <https://www.mpi-hd.mpg.de/hfm/CTA/MC/Software/Testing/>`_
+(CTA password applies) and install the package with:
 
+.. code-block:: console
 
-+++++++++++++++++++++++++++
-Set environmental variables
-+++++++++++++++++++++++++++
+    $ tar -czf corsika7.7_simtelarray.tar.gz
+    $ ./build_all prod5 qgs2 gsl
 
-Source the ``set_DB_environ.sh`` script (see `Model Database Access`_) to activate set the environmental variables for the DB access:
+Source the ``set_DB_environ.sh`` script (see `Model Database Access`_) to set the environmental variables required for database access:
```

Comment:
Do we plan to keep both the `set_DB_environ.sh` and the `.env` file? I thought we move completely to `.env`.